package.json

```diff
@@ -45,67 +50,77 @@
     "release:major": "node scripts/release/release.js major",
     "release:minor": "node scripts/release/release.js minor",
     "release:patch": "node scripts/release/release.js patch",
-    "test": "npm run jest && npm run compile && npm run nightwatch && npm run nightwatch:port-provided && wdio wdio.conf.js",
-    "wdio": "wdio wdio.conf.js"
+    "test": "npm run jest && npm run nightwatch && npm run nightwatch:port-provided && npm run wdio",
+    "wdio": "npm run wdio-webpack-obj && npm run wdio-webpack-func && npm run tt-wdio-webpack-obj && npm run tt-wdio-webpack-func",
+    "wdio-webpack-obj": "wdio",
+    "wdio-webpack-func": "TT_TEST_WDIO_FUNCTION=true wdio",
+    "tt-wdio-webpack-obj": "node scripts/wdio/wdio-runner-cli.js",
+    "tt-wdio-webpack-func": "TT_TEST_WDIO_FUNCTION=true node scripts/wdio/wdio-runner-cli.js"
   },
   "dependencies": {
     "async": "^2.5.0",
-    "autoprefixer": "^6.7.7",
-    "axe-core": "^2.6.1",
+    "autoprefixer": "^8.5.2",
```

Comment:
@bjankord @mjhenkes @ryanthemanuel @JakeLaCombe One thing to note with bumping "autoprefixer", when run compile:prod, the `css-loader` will be used to minify our css and this uses `cssnano`, which pulls in the autoprefixer version 6.7.7.... Realized some dependency difference on https://github.com/cerner/terra-clinical/pull/358 (long post there). Just to be aware or possible wait on?

Reply:
I'm ok with keeping autoprefix on `^6.0.0` until cssnano v4 is released. I think it would be good to reach out to the maintainers on that and see if we can get an idea when it may be released. 

I've run into another bug today with cssnano around how it minifies long hand CSS properties that are used with css custom properties. The "fix" was to turn `minimize` off in the css-loader options. Seeing as cssnano is causing issues now in multiple places, I'd like to discuss disabling the minimize option that triggers cssnano in our webpack config until cssnano v4 is released and consumed in the css-loader. The sass-loader already does a decent job of minification when webpack is run in prod mode. I can log an issue to kick off the discussion.

Reply:
It seems they've been working on v4 for some time. https://github.com/cssnano/cssnano/issues/408

Reply:
Talked offline. The consensus was to keep the autoprefixer bump and disable minification via the css-load until cssnano v4 is released